### ISSUE TITLE
How the code recognizes validation set for TextVQA

### ISSUE THREAD

**Issue by soonchangAI** [facebookresearch/mmf#1246]
## ❓ Questions and Help
Hi, I wanted to generate LMDB for new dataset. I noticed for TextVQA, the currently available LMDB stores training and validation set features using keys with prefix 'train/...' ; whereas, for test set, it uses keys with prefix "test/..."

I wonder during training and evaluation, how the code differentiates the validation set from training set since they share the same prefix for their keys. Plus,  <code>defaults.yaml</code>, does not contain location to IMDB files such as "imdb_val.npy" etc.

**Comment by soonchangAI** [comment]
dataset config

**Comment by soonchangAI** [comment]
dataset config
